aten/src/ATen/native/cuda/Math.cuh

```diff
@@ -88,5 +88,19 @@ static inline __host__ __device__ scalar_t calc_trigamma(scalar_t in) {
   return static_cast<scalar_t>(sign * result);
 }
 
+template <typename scalar_t>
+static inline __host__ __device__ scalar_t calc_gcd(scalar_t a, scalar_t b) {
+  using accscalar_t = at::acc_type<scalar_t, /*is_cuda=*/true>;
```

Comment:
Is the accumulate type ever different for the integer types?

Reply:
I thought `accscalar_t` stood for "accelerated scalar type" (as in accelerated with CUDA), my bad. I'll change that.